tests/VerificationClassTest.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+// PHPUNITが実行される場所がカレントディレクトリになるので要注意
+require_once './Verification/VerificationClass.php';
+
+class VerificationClassTest extends PHPUnit_Framework_TestCase {
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Opening brace of a class must be on the line after the definition